Park-Swift/screens/addressmapping.js

```diff
@@ -0,0 +1,78 @@
+// import { View, Text, StyleSheet} from 'react-native';
+import React, { useState, useEffect } from 'react';
+import { View, Image, Text, StyleSheet} from 'react-native';
+import axios from 'axios';
```

Comment:
Do we mean to include axios here? I see there is code commented out below that uses axios. Ideally, firebase will eliminate the need for axios to make api calls.

Reply:
I have added `npm install axios` to the install script